### PR TITLE
DAYMAN, AH AH AH! FIGHTER OF THE NIGHTMAN, AH AH AH! CHAMPION OF THE SUN!

### PR DESCRIPTION
**Pull request by Iamgoofball**
## About The Pull Request
khYTiAU

## Why It's Good For The Game
Adds emergent gameplay to spray paint cans.

Supplementary video:
https://www.youtube.com/watch?v=TzaVd6zl2bA

### CODE REVIEW

**Comment by Mothblocks** [comment]
Please consider this a friendly warning, your PR's must have a properly descriptive title and body content that explains the purpose of the PR.

Your current title and or body content does not meet requirements, so please update it.

**Comment by Mothblocks** [comment]
also you need a changelog

**Comment by tralezab** [comment]
Seems like a good addition but really lazily implemented.

**Comment by Iamgoofball** [comment]
what more do you want, you huff paint you get high and take brain damage, i dunno what to tell you man

File: code/game/objects/items/crayons.dm [hunk 1]
```diff
@@ -739,6 +739,12 @@
 		user.visible_message("<span class='danger'>[user] sprays [src] into the face of [target]!</span>")
 		to_chat(target, "<span class='userdanger'>[user] sprays [src] into your face!</span>")
 
+		if(user == target) // huffin spray
```
Comment (ATH1909) [inline code/game/objects/items/crayons.dm:742]: should probably also check to make sure that you can breathe the stuff in

**Comment by maxymax13** [comment]
> 
> 
> Seems like a good addition but really lazily implemented.

i mean, there's only so many ways to code paint huffing

File: code/game/objects/items/crayons.dm (hunk 1 above)
Comment (YakumoChen) [reply code/game/objects/items/crayons.dm:742]: make it require target the mouth so you can still spray your own face if you want

Comment (Iamgoofball) [reply code/game/objects/items/crayons.dm:742]: you're spraying paint onto your unprotected face, you're gonna get high off it

**Comment by Mothblocks** [comment]
> what more do you want

a properly descriptive title and body content that explains the purpose of the PR.

**Comment by Mothblocks** [comment]
do it

**Pull request closed without merging**